static/app/views/insights/pages/platform/shared/issuesWidget.tsx

```diff
@@ -302,15 +304,25 @@ const PanelContainer = styled(Panel)`
 `;
 
 const SuperHeaderLabel = styled(IssueStreamHeaderLabel)`
+  color: ${p => p.theme.headingColor};
+  font-size: 1rem;
+  line-height: 1.2;
   padding-left: ${space(1)};
   text-transform: capitalize;
+  font-weight: ${p => p.theme.fontWeightBold};
 `;
 
 const SuperHeader = styled(PanelHeader)`
-  background: ${p => p.theme.background};
+  background-color: ${p => p.theme.headerBackground};
   padding: ${space(1)};
 `;
 
+const HeaderContainer = styled('div')`
+  position: sticky;
+  top: 0;
+  z-index: 1001;
```

Comment:
why `1001`? that would be equal to a dropdown.
If possible reuse the definitions from the theme.
I think `zIndex.header` would be enough.
```suggestion
  z-index: ${p => p.theme.zIndex.header};;
```

Reply:
Ha, that was a leftover from the time I had two sticky's 😅